### PR TITLE
mail-client/alpine: fix file name

### PR DESCRIPTION
**Pull request by robert7k**
Closes: https://bugs.gentoo.org/666490

### CODE REVIEW

File: mail-client/alpine/Manifest [hunk 1]
```diff
@@ -1,3 +1,4 @@
+DIST 2.21.1-r1.tar.gz 7490140 BLAKE2B 13ac5ef43dd729a7cf21b44dcd6feace3e1813672aec537fd05d5ff90b67bec145ceef71cfcf35b0cb7e250bb307556f5dcf8ebb024bc6366fd0ace0a7c41161 SHA512 106caa7bfe86fd5e82554ba68b9ee579cd8495bcd0dfbf82d9d75da2df3bd292d42d67af8ec737091f2a820b2555c25bbf0058c9c7bb5de5f950b570ffc25c4d
```
Comment (mgorny) [inline mail-client/alpine/Manifest]: That's not a friendly or safe distfile name. It's asking for accidental collisions.

Comment (robert7k) [reply mail-client/alpine/Manifest]: Thanks for the hint, I fixed that.

File: mail-client/alpine/alpine-2.21.1-r1.ebuild [hunk 2]
```diff
@@ -7,8 +7,7 @@ inherit autotools toolchain-funcs vcs-snapshot
 DESCRIPTION="An easy to use text-based based mail and news client"
 HOMEPAGE="http://www.washington.edu/alpine/ http://repo.or.cz/alpine.git/"
 GIT_COMMIT="843b2f16abfd949e09b1c5465387b1b0f724994a"
-SRC_URI="http://repo.or.cz/alpine.git/snapshot/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
-RESTRICT="mirror"
+SRC_URI="http://repo.or.cz/alpine.git/snapshot/${GIT_COMMIT}.tar.gz -> ${PF}.tar.gz"
```
Comment (mgorny) [inline mail-client/alpine/alpine-2.21.1-r1.ebuild]: PF is no good here. It means that when someone revbumps it (e.g. due to dep change), he'll suddenly change filename for no reason.

Also, please match the upstream name, i.e. `alpine-843b2f1` (grab it off `${GIT_COMMIT::n}` where *n* is some number), and use that both for `->` name and `S`; then remove vcs-snapshot as that eclass is deprecated.

Comment (robert7k) [reply mail-client/alpine/alpine-2.21.1-r1.ebuild]: Thanks for pointing this out, I fixed it accordingly.

**Pull request closed without merging**